src/components/ErrorPage.js

```diff
@@ -0,0 +1,91 @@
+import { Component } from "react";
```

Comment:
please have a look on react hooks https://reactjs.org/docs/hooks-intro.html for your own knowledge